### ISSUE TITLE
nodes from groups of groups

### ISSUE THREAD

**Issue by mariuspana** [rundeck-plugins/ansible-plugin#31]
Hi,

We make pretty heavy use of groups of groups in our ansible inventory files. I just noticed that if we use something like the following in our /etc/ansible/hosts file our nodes list in rundeck does not return any nodes.

[group1]
nodeaaa

[group2]
nodebbb

[biggroup:children]
group1
group2

Im running rundeck2.6.7 on smartos. Is this known or am have I missed something?

Thanks

**Comment by frozenice** [comment]
Hm, can't seem to reproduce it with:
- Debian 7
- Rundeck 2.6.4-1 (edit: just upgrade to 2.6.7-1 and it still works)
- Ansible 2.0.1.0
- rundeck-ansible-plugin 1.2.4

I tried your exact config (replaced nodeaaa and nodebbb with actual nodes I have). It imported the two nodes and both have two tags (biggroup and group1 resp. group2).

Try enabling debug mode (see bottom of README) and look for a hint in the logs.

**Comment by mariuspana** [comment]
I failed to test my own example. I apologise for that.
My issue seems to have stemmed from the fact that I had ssh-agent enabled and for reasons I cannot explain, ansible would no longer connect to most of our hosts which left me with a weird nodes inventory in rundeck. I assumed that it was because of my nested groups.

Thank you for your help.

On 08 Jun 2016, at 20:45, David Kirstein <notifications@github.com<mailto:notifications@github.com>> wrote:

Hm, can't seem to reproduce it with:
-   Debian 7
-   Rundeck 2.6.4-1
-   Ansible 2.0.1.0
-   rundeck-ansible-plugin 1.2.4

I tried your exact config (replaced nodeaaa and nodebbb with actual nodes I have). It imported the two nodes and both have two tags (biggroup and group1 resp. group2).

Try enabling debug mode (see bottom of README) and look for a hint in the logs.
